packages/patrol_cli/lib/src/android/android_test_backend.dart

```diff
@@ -186,9 +188,44 @@ class AndroidTestBackend {
   Future<void> execute(
     AndroidAppOptions options,
     Device device, {
+    String? flavor,
     bool interruptible = false,
+    required bool showFlutterLogs,
+    required bool showTestSteps,
   }) async {
     await _disposeScope.run((scope) async {
+      // Read patrol logs from logcat
+      final processLogcat = await _processManager.start(
+        [
+          'adb',
```

Comment:
We should use `adb` commands using a wrapper. It's already available in the scope of this class